src/olympia/reviews/helpers.py

```diff
@@ -91,6 +93,15 @@ def user_can_delete_review(request, review):
             acl.action_allowed(request, 'Addons', 'Edit')))
 
 
+class CanDeleteReviewPermission(BasePermission):
+    """A DRF permission class wrapping user_can_delete_review()."""
+    def has_permission(self, request, view):
+        return self.request.user.is_authenticated()
+
+    def has_object_permission(self, request, view, obj):
+        return user_can_delete_review(request, obj)
+
```

Comment:
I'd rather see that class in `olympia.reviews.permissions` - `helpers` is mostly used for jinja/template related code so I don't feel comfortable having this here. What do you mean?